app/src/db/migrations/20240925201114_049-form-metadata.js

```diff
@@ -0,0 +1,27 @@
+const stamps = require('../stamps');
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+exports.up = function (knex) {
+  return Promise.resolve().then(() =>
+    knex.schema.createTable('form_metadata', (table) => {
+      table.uuid('id').primary();
+      table.uuid('formId').references('id').inTable('form').notNullable().index();
```

Comment:
Will we ever allow multiple metadata rows for a single formId? The concern here would be that a bug in the frontend creates multiples but it's non-deterministic which one is retrieved.

Reply:
no, one per form id.